src/user-search/lists.ts

```diff
@@ -0,0 +1,26 @@
+import {users as  badActorsList} from '../../user-data/lists/bad_actors/users'
+import {users as exchangesList } from '../../user-data/lists/exchanges/users'
+import {users as gdprList } from '../../user-data/lists/gdpr/users'
+// import * as verifiedList from '../../user-data/lists/verified/users' FIXME
+
+enum UserAccountTags {
```

Comment:
not used

Reply:
`verified` is in the spec as a possible user tag, but I can remove this until we have at least one user who is `verified`